### PR TITLE
Add translations for Igbo, Yoruba & Pidgin error pages

### LINKED ISSUE #1947: Add translations for error pages for Igbo, Yoruba and Pidgin
**Is your feature request related to a problem? Please describe.**
In https://github.com/bbc/simorgh/issues/1887 we added translations for the cookie/privacy banners. We also need to add translations for error pages from the translations spreadsheet.

**Describe the solution you'd like**
Add translations for all text required on error pages for Pidgin, Yoruba and Igbo.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Storybook, General regression of news & persian article 404 pages

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #1947 

**Overall change:** _Adds translations for Igbo, Yoruba & Pidgin error pages._

**Code changes:**

- _Add translations for Igbo, Yoruba & Pidgin error pages._
- _Add stories for Igbo, Yoruba & Pidgin error pages ._
- _Update containers/FrontPage test._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/lib/config/services/igbo.js [hunk 1]
```diff
@@ -33,32 +33,32 @@ const igbo = {
...
+        callToActionLast: '.',
+        callToActionLinkUrl: 'https://www.bbc.com/igbo',
       },
       500: {
         statusCode: '500',
-        title: 'Internal server error',
+        title: 'Mperi',
         message:
-          "Sorry, we're currently unable to bring you the page you're looking for. Please try:",
+          'E wela iwe, anyị enweghị ike iwetara gị ibe akwụkwọ ị na-achọ.Gbalịa ọzọ:',
```
Comment (andrew-nowak) [inline src/app/lib/config/services/igbo.js]:
```suggestion
          'E wela iwe, anyị enweghị ike iwetara gị ibe akwụkwọ ị na-achọ. Gbalịa ọzọ:',
```

pretty sure this is a typo in the translations spreadsheet cc @clareevansbbc

File: src/app/lib/config/services/pidgin.js [hunk 2]
```diff
@@ -33,32 +33,32 @@ const pidgin = {
...
-          'Hitting the refresh button in your browser',
-          'Coming back again later',
+          'Press di refresh button for your browser',
+          'To come back again afta now',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'BBC News homepage.',
-        callToActionLast: '',
-        callToActionLinkUrl: 'https://www.bbc.com/news',
+        callToActionFirst: 'Or, abeG visit di ',
```
Comment (andrew-nowak) [inline src/app/lib/config/services/pidgin.js]:
```suggestion
        callToActionFirst: 'Or, abeg visit di ',
```

almost definitely a translation typo

File: src/app/lib/config/services/pidgin.js [hunk 3]
```diff
@@ -33,32 +33,32 @@ const pidgin = {
...
-          'Searching for this page using the BBC search bar',
+          'Check di url again',
+          'Press di refresh button for your browser',
+          'Search for dis page using di BBC search bar',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'BBC News homepage.',
-        callToActionLast: '',
-        callToActionLinkUrl: 'https://www.bbc.com/news',
+        callToActionFirst: 'Or, abeG visit di ',
```
Comment (andrew-nowak) [inline src/app/lib/config/services/pidgin.js]:
```suggestion
        callToActionFirst: 'Or, abeg visit di ',
```

**Review by j-pendlebury — APPROVED** [review]
Just needs the fixes Andrew spotted. Otherwise LGTM 👍

File: src/app/lib/config/services/igbo.js (hunk 1 above)
Comment (Bopchy) [reply src/app/lib/config/services/igbo.js]: Also pretty sure that it's a typo. Went ahead and committed it as it's just adding a space

**Review by andrew-nowak — APPROVED** [review]

**Comment by david-boydell** [comment]
Sorry, to be a pain @Bopchy. Please could we have a full stop after the last sentence for `callToActionLast: '',` in `src/app/lib/config/services/yoruba.js ` for the 404 and the 500?

**Comment by david-boydell** [comment]
Once done I'll update the spreadsheet

**Comment by david-boydell** [comment]
Thanks @Bopchy, this is ready for merge.

**Pull request merged**